packages/datagrid/src/basicmousehandler.ts

```diff
@@ -613,8 +613,8 @@ class BasicMouseHandler implements DataGrid.IMouseHandler {
     return this._pressData;
   }
 
-  private _disposed = false;
-  private _pressData: PressData.PressData | null = null;
+  protected _disposed = false;
+  protected _pressData: PressData.PressData | null = null;
```

Comment:
Any inheriting class should be able to use the isDisposed/pressData methods, no?

Reply:
I mean, I understand why an inheritor might want to *write* to _pressData, but the `_disposed` property should not need to be directly written to ?

Reply:
You are right - fixed now.